ping_examples/example-all-metrics.yaml

```diff
@@ -0,0 +1,99 @@
+metrics:
```

Comment:
Can you clarify, will all the possible probe types be broken out in the ping, as is shown here? I wasn't sure if we were doing it that way, or if we would represent everything with some basic probe types (e.g. a gauge is just a keyed int scalar), but I think I like it this way more.

Reply:
I think this is the way we want to move forward.  It will require more changes and support on the backend pipeline, however, so I should probably chat with @fbertsch, @mreid-moz and others about the feasibility of those changes.